### DISCUSSION TITLE
I think this is a typo

### DISCUSSION THREAD

**Discussion by dongho-jung** [actions/actions-runner-controller#1941] [General]
In main.go, the default value of syncPeriod is 1min
https://github.com/actions-runner-controller/actions-runner-controller/blob/5298c6ea298152d7b6422f3c75e83ed9ea019572/main.go#L118

and also in detailed-docs.md, it's 1min
https://github.com/actions-runner-controller/actions-runner-controller/blob/5298c6ea298152d7b6422f3c75e83ed9ea019572/docs/detailed-docs.md?plain=1#L566

but in charts README.md, it's 10min
https://github.com/actions-runner-controller/actions-runner-controller/blob/5298c6ea298152d7b6422f3c75e83ed9ea019572/charts/actions-runner-controller/README.md?plain=1#L16

and I also think it should be controller not controler

I was gonna repot this in GitHub Issues though, it seemed that it's too much to this

**Comment by toast-gear** [comment]
yeh these are mistakes, originally the default was 10 mins  back in the day but we adjusted it to 1 minute quite a while ago once caching was introduced. Please do raise a PR to correct the docs, same for the spelling fix from controler to controller

**Reply by dongho-jung** [reply]
ok I issued PR [here](https://github.com/actions-runner-controller/actions-runner-controller/pull/1942) :-)
